CustomeDB/LinkedList.cpp

```diff
@@ -54,21 +96,21 @@ Node* LinkedList::getclientHead()
 
 void LinkedList::addAtHead(string name, int id, double salary, string project, string joiningDate, string phone, string address)
 {
-	Node* new_node = new Node(name, id, salary, project, joiningDate, phone, address);
-	new_node->next = NULL;
-
-	if (emphead == NULL) {
-		emphead = new_node;
-	}
-	else {
-		Node* temp = emphead;
-		emphead = new_node;
-		emphead->next = temp;
-	}
+    Node* new_node = new Node(name, id, salary, project, joiningDate, phone, address);
+    new_node->next = NULL;
+
+    if (emphead == NULL) {
+        emphead = new_node;
+    }
+    else {
+        Node* temp = emphead;
+        emphead = new_node;
+        emphead->next = temp;
+    }
 }
 
-void LinkedList::addAtHead(string name, int id, string phone, string address, int project_id) {
-    Node* new_node = new Node(name, id, phone, address, project_id);
+void LinkedList::addAtHead(string name, int cl_id, string phone, string address, int project_id) {
```

Comment:
do not create overloaded functions. just name them differently. there are other such inconsistencies, resolve them all